Support fixed database role db_accessadmin

tanscorpio7 opened the pull request:
### Description

[Describe what this change achieves]
 
#### Engine PR: https://github.com/babelfish-for-postgresql/postgresql_modified_for_babelfish/pull/447
#### Extension PR: https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/2970

### Issues Resolved

[List any issues this PR will resolve]
 
### Check List

- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is under the terms of the PostgreSQL license, and grant any person obtaining a copy of the contribution permission to relicense all or a portion of my contribution to the PostgreSQL License solely to contribute all or a portion of my contribution to the PostgreSQL open source project.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/babelfish-for-postgresql/postgresql_modified_for_babelfish/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

tanscorpio7 committed:
dump restore changes for db_accessadmins fixed db role

Signed-off-by: Tanzeel Khan <tzlkhan@amazon.com>

tanscorpio7 committed:
Merge remote-tracking branch 'origin/BABEL_4_X_DEV__PG_16_X' into BABEL-5136

tanscorpio7 committed:
Merge remote-tracking branch 'origin/BABEL_4_X_DEV__PG_16_X' into BABEL-5136

tanscorpio7 committed:
Merge remote-tracking branch 'origin/BABEL_4_X_DEV__PG_16_X' into BABEL-5136

tanscorpio7 committed:
Merge remote-tracking branch 'origin/BABEL_4_X_DEV__PG_16_X' into BABEL-5136

tanscorpio7 committed:
handle dump restore for db_accessadmin acls

Signed-off-by: Tanzeel Khan <tzlkhan@amazon.com>

tanscorpio7 committed:
fix

Signed-off-by: Tanzeel Khan <tzlkhan@amazon.com>

rishabhtanwar29 commented on src/bin/pg_dump/dump_babel_utils.c:
```diff
@@ -1124,9 +1124,9 @@ addFromClauseForLogicalDatabaseDump(PQExpBuffer buf, TableInfo *tbinfo)
-						  "('master_dbo', 'master_db_owner', 'master_guest', "
-						  "'msdb_dbo', 'msdb_db_owner', 'msdb_guest', "
-						  "'tempdb_dbo', 'tempdb_db_owner', 'tempdb_guest') ",
+						  "('master_dbo', 'master_db_owner', 'master_guest', 'master_db_accessadmin', "
+						  "'msdb_dbo', 'msdb_db_owner', 'msdb_guest', 'msdb_db_accessadmin', "
+						  "'tempdb_dbo', 'tempdb_db_owner', 'tempdb_guest', 'tempdb_db_accessadmin') ",
```
Would appreciate if you can refactor these default role to a static array like we have `default_bbf_roles` in dumpall_babel_utils.c

tanscorpio7 replied:
Done

tanscorpio7 committed:
refactor

Signed-off-by: Tanzeel Khan <tzlkhan@amazon.com>

anju15bharti committed:
[Babelfish] Do not dump securityadmin from BBF catalog (#95)

Description 
Babel: Do not dump securityadmin from BBF catalog

Extension PR https://github.com/amazon-aurora/babelfish_extensions/pull/67

Task: BABEL-5040
Signed-off-by: ANJU BHARTI <abanju@amazon.com>

tanscorpio7 committed:
Merge remote-tracking branch 'amazon/server-database-roles-4-x' into BABEL-5136

tanscorpio7 commented:
https://github.com/amazon-aurora/postgresql_modified_for_babelfish/pull/96
New PR
